arviz/data/io_pymc3.py

```diff
@@ -1,29 +1,54 @@
 """PyMC3-specific conversion code."""
 import logging
 from typing import Dict, List, Any, Optional, TYPE_CHECKING
+from types import ModuleType
 
 import numpy as np
 import xarray as xr
 from .. import utils
-from .inference_data import InferenceData
+from .inference_data import InferenceData, concat
 from .base import requires, dict_to_dataset, generate_dims_coords, make_attrs
 
 if TYPE_CHECKING:
     import pymc3 as pm
+    from pymc3 import MultiTrace, Model  # pylint: disable=invalid-name
+    import theano
+    from typing import Set  # pylint: disable=ungrouped-imports
+else:
+    MultiTrace = Any  # pylint: disable=invalid-name
+    Model = Any  # pylint: disable=invalid-name
+
+___all__ = [""]
 
 _log = logging.getLogger(__name__)
 
 Coords = Dict[str, List[Any]]
 Dims = Dict[str, List[str]]
+# random variable object ...
+Var = Any  # pylint: disable=invalid-name
+
+# pylint: disable=line-too-long
```

Comment:
What is the scope of this diasable? (mainly curiosity)

Reply:
> What is the scope of this disable? (mainly curiosity)

I'm afraid I don't know as well as I should.  I'm going to try to leave it out and see what happens.  If it's not necessary (after black reformatting), or if it can be more tightly scoped, I will cut or mve it.

Reply:
I cut it out -- I think it was there because my editor didn't look at the right .pylintrc, and was holding me to 80 characters.  Passes pylint here without it.  Assuming it passes in CI, I think we are good.